rotkehlchen/chain/evm/constants.py

```diff
@@ -5,4 +5,24 @@
 MAX_BLOCKTIME_CACHE = 250  # 55 mins with 13 secs avg block time
 ZERO_ADDRESS = string_to_evm_address('0x0000000000000000000000000000000000000000')
 ETH_SPECIAL_ADDRESS = string_to_evm_address('0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE')
-GENESIS_HASH = deserialize_evm_tx_hash('0x' + '0' * 64)  # hash for transactions in genesis block
+ZERO_32_BYTES_HEX = '0x' + '0' * 64
+GENESIS_HASH = deserialize_evm_tx_hash(ZERO_32_BYTES_HEX)  # hash for transactions in genesis block
+
+# Fake receipt with values taken from ethereum mainnet, to emulate a receipt for the
+# genesis transactions
+FAKE_GENESIS_TX_RECEIPT = {  # Uses ethereum receipt structure
```

Comment:
Please apply [my comment](https://github.com/rotki/rotki/pull/5616#discussion_r1108802070) directly. If you notice this comment is now redundant
```suggestion
FAKE_GENESIS_TX_RECEIPT = {
```